clinics/merge/clinicians.go

```diff
@@ -56,6 +60,16 @@ func (c ClinicianPlans) PreventsMerge() bool {
 	return PlansPreventMerge(c)
 }
 
+func (c ClinicianPlans) PendingInvitesByWorkspace() map[string]int{
+	result := make(map[string]int)
+	for _, p := range c {
+		if p.IsPendingInvite() {
+			result[p.Workspaces[0]] = result[p.Workspaces[0]] + 1
```

Comment:
Can't a `ClinicianPlan` contain more than one workspace? Should those be handled?

Reply:
A pending invite is bound to a single workspace